core/modules/comment/comment.module

```diff
@@ -2018,7 +2018,7 @@ function comment_preview($comment) {
       $comment->picture = $account->picture;
     }
     elseif (empty($comment->name)) {
-      $comment->name = config_get('system.core', 'anonymous');
+      $comment->name = config_get_translated('system.core', 'anonymous');
```

Comment:
```suggestion
      $comment->name = config_get('system.core', 'anonymous');
```

Reply:
Thanks @quicksketch I've updated the PR with all your suggestions.